src/main/java/com/spotify/docker/client/messages/HostConfig.java

```diff
@@ -46,6 +46,11 @@
   @JsonProperty("VolumesFrom") private ImmutableList<String> volumesFrom;
   @JsonProperty("NetworkMode") private String networkMode;
   @JsonProperty("SecurityOpt") private ImmutableList<String> securityOpt;
+  @JsonProperty("Memory") private Long memory;
+  @JsonProperty("MemorySwap") private Long memorySwap;
+  @JsonProperty("CpuShares") private Long cpuShares;
+  @JsonProperty("CpusetCpus") private String cpuset;
```

Comment:
Any reason to use `cpuset` instead of `cpisetCpus`? The latter would be more consistent with the docker API.